tmk_core/common/keycode.h

```diff
@@ -46,6 +46,22 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define MOD_BIT(code)            (1 << MOD_INDEX(code))
 #define MOD_INDEX(code)          ((code) & 0x07)
 
+#define MOD_MASK_CTRL            (MOD_BIT(KC_LCTRL)  | MOD_BIT(KC_RCTRL))
+#define MOD_MASK_SHIFT           (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT))
+#define MOD_MASK_ALT             (MOD_BIT(KC_LALT)   | MOD_BIT(KC_RALT))
+#define MOD_MASK_GUI             (MOD_BIT(KC_LGUI)   | MOD_BIT(KC_RGUI))
+#define MOD_MASK_CS              (MOD_MASK_CTRL  | MOD_MASK_SHIFT)
+#define MOD_MASK_CA              (MOD_MASK_CTRL  | MOD_MASK_ALT)
+#define MOD_MASK_CG              (MOD_MASK_CTRL  | MOD_MASK_GUI)
+#define MOD_MASK_SA              (MOD_MASK_SHIFT | MOD_MASK_ALT)
+#define MOD_MASK_SG              (MOD_MASK_SHIFT | MOD_MASK_GUI)
+#define MOD_MASK_AG              (MOD_MASK_ALT   | MOD_MASK_GUI)
+#define MOD_MASK_CSA             (MOD_MASK_CTRL  | MOD_MASK_SHIFT | MOD_MASK_ALT)
+#define MOD_MASK_CSG             (MOD_MASK_CTRL  | MOD_MASK_SHIFT | MOD_MASK_GUI)
+#define MOD_MASK_CAG             (MOD_MASK_CTRL  | MOD_MASK_ALT   | MOD_MASK_GUI)
+#define MOD_MASK_SAG             (MOD_MASK_SHIFT | MOD_MASK_ALT   | MOD_MASK_GUI)
+#define MOD_MASK_ALL             (MOD_MASK_CTRL  | MOD_MASK_SHIFT | MOD_MASK_ALT | MOD_MASK_GUI)
```

Comment:
Should be: 
```suggestion
#define MOD_MASK_HYPR             (MOD_MASK_CTRL  | MOD_MASK_SHIFT | MOD_MASK_ALT | MOD_MASK_GUI)
```

As that's what we

Reply:
I can definitely add aliases for this, but I'm not sure `tmk_core` is the right place for it, as the TMK code doesn't actually know about any of those (they're a strictly QMK thing). Separation of concerns applies.
Maybe I can put the aliases in `quantum_keycodes.h`?

Also, the existing names aren't consistent either.
https://github.com/qmk/qmk_firmware/blob/5f11426d2c1676e9cdd0bc13418057e945672c95/quantum/quantum_keycodes.h#L475
https://github.com/qmk/qmk_firmware/blob/5f11426d2c1676e9cdd0bc13418057e945672c95/quantum/quantum_keycodes.h#L661

Reply:
The first on is, the HYPR, but the other ... yeah. 

But I'd feel better if the were all consistent.

Reply:
So would I 😕 That's why, for now, I'm going to feign ignorance and totally forget those exist, and just use what makes the most logical sense for `tmk_core`.